src/Harbor.Tagd/Args/CommonArgs.cs

```diff
@@ -10,6 +10,9 @@ public class CommonArgs
 		[NamedArgument('v', "verbosity", Description = "How verbose should logging output be")]
 		public string Verbosity { get; set; } = "info";
 
+		[NamedArgument("timeout", Description = "Timeout in seconds for http requests, defaults to 600")]
+		public double Timeout { get; set; } = 600;

```

Comment:
What's the reasoning behind making this a double instead of an int or a long?

Reply:
TimeSpan.FromSeconds takes a double - I am not a C# developer so wasn't too sure on how type casting would work/apply.

Reply:
No worries. Let's go ahead and make this an `int`. I don't think it makes sense to have a fractional-second timeout:

```suggestion
		public int Timeout { get; set; } = 600;
```